PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellEnergy.cxx

```diff
@@ -56,6 +57,9 @@ Bool_t AliEmcalCorrectionCellEnergy::Initialize()
   // check the YAML configuration if the Run2 calibration is requested (default is false)
   GetProperty("enableNewTempCalib",fUseNewRunDepTempCalib);
 
+  // check the YAML configuration if temperature calibration is disabled explicitly (default is false)
+  GetProperty("disableTempCalib",fDisableTempCalib);
```

Comment:
This needs a default value in the base config

Reply:
done.